Elements/src/Model.cs

```diff
@@ -245,6 +271,12 @@ public static Model FromJson(string json, bool forceTypeReload = false)
         }
 
         private List<Element> RecursiveGatherSubElements(object obj)
+        {
+            var props = new Dictionary<Type, List<PropertyInfo>>();
```

Comment:
This is a new dictionary created for the purpose of caching properties that we need to recurse, for types that we've seen before.